crates/stark_hash/src/serde.rs

```diff
@@ -52,17 +52,17 @@ mod tests {
     fn empty() {
         assert_eq!(
             serde_json::from_str::<StarkHash>(r#""""#).unwrap(),
-            StarkHash::ZERO
+            StarkHash::default()
```

Comment:
A question for @CHr15F0x - why should an empty string `""` be a valid hash value? Does that actually come into play somewhere or is that more a result of our parsing algo?

Reply:
I vaguely recall some discussion long time ago which started at `"0x"` is a valid hex value meaning a zero (hash), and in consequence since the prefix `0x` is optional then an empty string `""` is a valid value (all bytes `0`).